src/libtest/lib.rs

```diff
@@ -462,10 +653,28 @@ fn optgroups() -> getopts::Options {
             unstable-options = Allow use of experimental features",
             "unstable-options",
         )
-        .optflag(
+        .optflagopt(
             "",
             "report-time",
-            "Show execution time of each test. Not available for --format=terse"
+            "Show execution time of each test. Awailable values:
+            plain   = do not colorize the execution time (default);
+            colored = colorize output according to the `color` parameter value;
+
+            Threshold values for colorized output can be configured via
+            `RUST_TEST_TIME_UNIT`, `RUST_TEST_TIME_INTEGRATION` and
+            `RUST_TEST_TIME_DOCTEST` environment variables.
```

Comment:
Is there any message visible to the user explaining him that these environment variables contain the warn time and the critical time?

Reply:
It will be in the unstable book (and I will add a link to the corresponding page in that message) when somebody will answer [my question](https://github.com/rust-lang/rust/pull/64873#issuecomment-536254516).